force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -122,13 +201,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of all integers up to the limit.
      */
     public static Integer sumUpToLimit(Integer intLimit) {
-        /* 
-         for (){
-            
-         }
-        */
+        
+        Integer sum = 0;
 
-        return null; // Replace null with the variable you used to store the result
+        for (Integer i = 0; i < intLimit; i++) {
+            sum += (i + 1);
```

Comment:
You can also start your iteration from 1, so you dont need to add 1 to each i during each loop.

Reply:
I know... I'm old school and that just felt "dirty" to me lol, but ya I thought about it :-)